documentation/docs/25-build-and-deploy/70-adapter-cloudflare-workers.md

```diff
@@ -36,6 +36,14 @@ export default {
 
 Path to your custom `wrangler.toml` or `wrangler.json` config file.
 
+### handlers
+
+Path to a file with additional [handlers](https://developers.cloudflare.com/workers/runtime-apis/handlers/) and (optionally) [Durable Objects](https://developers.cloudflare.com/durable-objects/) to be exported from the file the adapter generates. This allows you to, for example, include handlers for scheduled or queue triggers alongside the fetch handler your SvelteKit app.
```

Comment:
I'm not sure the durable objects case works. They need to be named exports in the main worker file and no re-exported from another file.